coda_v2_python_client/firebase_client_wrapper.py

```diff
@@ -678,7 +678,14 @@ def get_dataset_user_ids(self, dataset_id, transaction=None):
         :rtype: list
         """
         self.ensure_user_ids_consistent(dataset_id, transaction=transaction)
-        return self.get_segment(dataset_id, transaction=transaction).get("users")
+
+        segment_snapshot = self.get_segment(dataset_id, transaction=transaction)
+        segment_doc = segment_snapshot.to_dict()
+
+        user_ids = []
+        if segment_doc is not None and "users" in segment_doc:
+            user_ids = segment_snapshot.get("users")
+        return user_ids
```

Comment:
If the document or users don't exist should we return None or []?

Reply:
updated the function to return None.